[github-actions] macOS build checks are failing

```
Run brew install automake m4 ninja
  brew install automake m4 ninja
  [ clang != clang ] || brew install llvm
  shell: /bin/bash -e {0}
  env:
    CC: clang
    CXX: clang++
Warning: m4 1.4.18 is already installed and up-to-date.
To reinstall 1.4.18, run:
  brew reinstall m4
==> Downloading https://homebrew.bintray.com/bottles/automake-1.16.3_1.catalina.bottle.tar.gz
curl: (22) The requested URL returned error: 403 Forbidden
Error: Failed to download resource "automake"
Download failed: https://homebrew.bintray.com/bottles/automake-1.16.3_1.catalina.bottle.tar.gz
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -261,6 +261,7 @@ jobs:
         submodules: true
     - name: Bootstrap
       run: |
+        brew update
         brew install automake m4 ninja
         [ ${{ matrix.CC }} != clang ] || brew install llvm
     - name: Build

--- a/.github/workflows/posix.yml
+++ b/.github/workflows/posix.yml
@@ -52,6 +52,7 @@ jobs:
     - uses: actions/checkout@v2
     - name: Bootstrap
       run: |
+        brew update
         brew install ninja
     - name: Run RCP Mode
       run: |
@@ -296,6 +297,7 @@ jobs:
     - uses: actions/checkout@v2
     - name: Bootstrap
       run: |
+        brew update
         brew install ninja socat
     - name: Build
       run: |

